apps/dapp-console-api/src/routes/contracts/ContractsRoute.ts

```diff
@@ -177,7 +184,9 @@ export class ContractsRoute extends Route {
               deployerAddress,
               chainId,
               appId,
-              state: ContractState.NOT_VERIFIED,
+              state: isDeployerVerified
```

Comment:
The FE reads the state of the contract after it's been created to decide if it needs to show the verification flow or just show a success screen